specification/storage/data-plane/Microsoft.FileStorage/stable/2025-01-05/file.json

```diff
@@ -328,19 +328,25 @@
                 "x-ms-client-name": "ShareProvisionedIops",
                 "type": "integer",
                 "format": "int64",
-                "description": "The provisioned IOPS of the share. If this is not specified, compute the recommended IOPS of the share using the formula for a share in this media tier (SSD/HDD as appropriate)."
+                "description": "The provisioned IOPS of the share."
               },
               "x-ms-share-provisioned-bandwidth-mibps": {
                 "x-ms-client-name": "ShareProvisionedBandwidthMibps",
                 "type": "integer",
                 "format": "int64",
-                "description": "The provisioned throughput of the share. If this is not specified, compute the recommended throughput of the share using the formula for a share in this media tier (SSD/HDD as appropriate)."
+                "description": "The provisioned throughput of the share."
               },
               "x-ms-share-included-burst-iops": {
                 "x-ms-client-name": "ShareIncludedBurstIops",
                 "type": "integer",
                 "format": "int64",
-                "description": "ShareIncludedBurstIops"
+                "description": "Returns the calculated burst IOPS of the share."
+              },
+              "x-ms-share-max-burst-credits-for-iops": {
+                "x-ms-client-name": "MaxBurstCreditsForIops",
+                "description": "Returned the calculated maximum burst credits. This is not the current burst credit level, but the maximum burst credits the share can have.",
```

Comment:
Returns the calculated maximum burst credits

Reply:
Fixed.